project/README.rst

```diff
@@ -1,46 +1,135 @@
-================
-NSD Term Project
-================
+PoissonNet
```

Comment:
You destroyed the project README.  Please fix it and remove bad commits from history (squashing and forcing push may be the easiest way to do it).